Client/mods/deathmatch/logic/CClientVectorGraphic.cpp

```diff
@@ -80,6 +80,9 @@ bool CClientVectorGraphic::RemoveUpdateCallback()
 
 void CClientVectorGraphic::OnUpdate()
 {
+    if (!m_pVectorGraphicDisplay)
```

Comment:
This pointer will reference to unloaded data.
m_pVectorGraphicDisplay must be std::shared_ptr, to prevent this.
You replaced unique_ptr to shared_ptr without benefits and memory safety rules